python/tests/integration/arcticdb/test_arctic_batch.py

```diff
@@ -0,0 +1,962 @@
+"""
+Copyright 2023 Man Group Operations Limited
+
+Use of this software is governed by the Business Source License 1.1 included in the file licenses/BSL.txt.
+
+As of the Change Date specified in that file, in accordance with the Business Source License, use of this software will be governed by the Apache License, version 2.0.
+"""
+import sys
+import os
+
+import pytz
+from arcticdb_ext.exceptions import InternalException, ErrorCode, ErrorCategory
+from arcticdb.exceptions import ArcticNativeNotYetImplemented, LibraryNotFound
+from pandas import Timestamp
+
+try:
+    from arcticdb.version_store import VersionedItem as PythonVersionedItem
+except ImportError:
+    # arcticdb squashes the packages
+    from arcticdb._store import VersionedItem as PythonVersionedItem
+from arcticdb_ext.storage import NoDataFoundException, KeyType
+from arcticdb_ext.version_store import DataError, VersionRequestType
+
+from arcticdb.arctic import Arctic
+from arcticdb.adapters.s3_library_adapter import S3LibraryAdapter
+from arcticdb.options import LibraryOptions
+from arcticdb.encoding_version import EncodingVersion
+from arcticdb import QueryBuilder
+from arcticc.pb2.s3_storage_pb2 import Config as S3Config
+
+import math
+import re
+import pytest
+import pandas as pd
+from datetime import datetime, date, timezone, timedelta
+import numpy as np
+from arcticdb_ext.tools import AZURE_SUPPORT
+from numpy import datetime64
+from arcticdb.util.test import (
+    assert_frame_equal,
+    random_strings_of_length,
+    random_floats,
+)
+import random
```

Comment:
any unused imports here?

Reply:
reviewed all of them, removed the non-used ones